source/macos/AdaptiveCards/AdaptiveCards/Views/ACRMultilineInputTextView.swift

```diff
@@ -71,7 +71,7 @@ class ACRMultilineInputTextView: NSView, NSTextViewDelegate {
         wantsLayer = true
         layer?.borderWidth = inputConfig.borderWidth
         layer?.cornerRadius = inputConfig.focusRingCornerRadius
-        textView.setAccessibilityTitle(config.localisedStringConfig.inputTextFieldAccessibilityTitle)
+        textView.setAccessibilityRoleDescription(config.localisedStringConfig.inputTextFieldAccessibilityTitle)
```

Comment:
Same here. 
Let's try to use the methods that it meant to be used for this

Reply:
RoleDescriptor is the method that is meant to be used for this as far as I understand. The strings are customized for that. For e.g. - ```inputTextFieldAccessibilityTitle``` returns the string ```Text Field```, which is the role of the field, not the title, right?

Reply:
Makes sense @snbehar !